src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java

```diff
@@ -496,8 +496,8 @@ public StreamRequestHandler handleRequest(final DataInputStream inputStream,
             for(String storeName: storeNames) {
 
                 ReadOnlyStorageEngine store = getReadOnlyStorageEngine(metadataStore,
-                                                                       storeRepository,
```

Comment:
Hi Arun,

Thanks for all your comments and sorry for my bad coding style. I also
found out I made mistake commits last time to my own github (that was for
concurrency push issues, haven't done yet) which made my commit timeline
mess up. I have re-forked Voldemort repo and will only commit codes for
invokeFetch() swallows and rethrow exception issue this time.


On Fri, Feb 26, 2016 at 4:29 PM, ARUNACHALAM THIRUPATHI <
notifications@github.com> wrote:

> In
> src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java
> <https://github.com/voldemort/voldemort/pull/377#discussion_r54321256>:
>
> > @@ -496,8 +496,8 @@ public StreamRequestHandler handleRequest(final DataInputStream inputStream,
> >              for(String storeName: storeNames) {
> >
> >                  ReadOnlyStorageEngine store = getReadOnlyStorageEngine(metadataStore,
> > -                                                                       storeRepository,
>
> Though it is OK for now, you should have an editor setting format only
> editted lines. It makes it harder to review on big changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/voldemort/voldemort/pull/377/files#r54321256>.
>